doctor/utils/__init__.py

```diff
@@ -3,6 +3,8 @@
 import re
 import sys
 
+import six
+from past.builtins import execfile
```

Comment:
I tried switching this to `with open(...): exec(compile(...), ...)`, but had some trouble with tests in mocking `exec`. I'll give it another shot tomorrow.